### PR TITLE
Site delete endpoint mq 30

### PR DESCRIPTION
**Pull request by joshplumley**

### CODE REVIEW

**Comment by jarrilla** [comment]
@joshplumley 
Hey sorry! I missed something on this one.

We should add a flag to the model.. something like `inactive: Boolean`... and all we wanna do is set that to true. This way we don't lose data from deleting sites unless we wanna do a purge later on

File: src/site/controller.ts [hunk 1]
```diff
@@ -11,9 +11,10 @@ import { verifySiteId } from './utils';
 const SiteRouter = Router();
 export default SiteRouter;
 
-SiteRouter.get(['/:siteId'], (res, req, next) =>
+SiteRouter.get(['/:siteId'], async (res, req, next) =>
```
Comment (jbolles94) [inline src/site/controller.ts:14]: Does this need to be async if the function called here doesn't have an await?

Comment (joshplumley) [reply src/site/controller.ts:14]: nope

File: src/site/controller.ts [hunk 2]
```diff
@@ -14,6 +14,9 @@ export default SiteRouter;
 SiteRouter.get(['/:siteId'], (req, res, next) =>
   checkId(res, next, SiteModel, req.params.siteId),
 );
+SiteRouter.delete(['/'], (req, res, next) =>
```
Comment (jbolles94) [inline src/site/controller.ts:17]: async/awaits

**Review by jbolles94 — APPROVED** [review]

**Pull request merged**